utils/build-presets.ini

```diff
@@ -1701,6 +1701,26 @@ assertions
 swiftsyntax
 swiftsyntax-verify-generated-files
 
+#===------------------------------------------------------------------------===#
+# Test Swift Format
+#===------------------------------------------------------------------------===#
+
+[preset: buildbot_swiftformat_macos]
+mixin-preset=mixin_swiftpm_package_macos_platform
+release
+assertions
+swiftsyntax
+skip-test-swiftsyntax
+swiftformat
+
+[preset: buildbot_swiftformat_linux]
+mixin-preset=mixin_swiftpm_package_linux_platform
+release
+assertions
+swiftsyntax
+skip-test-swiftsyntax
```

Comment:
Same here. IIRC there is no need to explicitly specify the dependency on SwiftSyntax.